### PR TITLE
DOC-12251 Update Power BI doc with new version of the ODBC driver

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 = Couchbase Power BI Connector
 :page-toclevels: 2
-:description: The Couchbase Power BI Connector provides the integration between tabular views of the Couchbase Analytics Service -- or your Capella columnar instance -- and Microsoft's Power BI interactive data visualization platform.
+:description: The Couchbase Power BI Connector integrates the tabular views of the Couchbase Analytics Service or your Capella columnar instance with Microsoft's Power BI interactive data visualization platform.
```
Comment (janhavitripurwar) [inline modules/ROOT/pages/index.adoc]: It also works with Capella Provisioned.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -55,7 +55,7 @@ OpenSSL is a required dependency for the driver.
 
 === Install OpenSSL
 
-. Download `Win64OpenSSL_Light-1_1_1v.msi` from the https://slproweb.com/products/Win32OpenSSL.html[Shining Light Productions site].
+. Download `Win64 OpenSSL v3.3.x Light` from the https://slproweb.com/products/Win32OpenSSL.html[Shining Light Productions site].
```
Comment (janhavitripurwar) [inline modules/ROOT/pages/index.adoc]: It should work with v3.3.x but QE has signed off with v3.3.1. Should we say this version is for [couchbase-odbc-1.1-win64.msi](https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-1.1-win64.msi)?

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -10,6 +10,16 @@ and either Couchbase Analytics Service (CBAS) or tabular views of your Capella c
 
 For self-managed CBAS, the Power BI connector requires Couchbase Server 7.2.4 or newer.
```
Comment (janhavitripurwar) [inline modules/ROOT/pages/release-notes.adoc:11]: for self-managed CBAS and also for Capella provisioned.

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -10,6 +10,16 @@ and either Couchbase Analytics Service (CBAS) or tabular views of your Capella c
...
+https://packages.couchbase.com/releases/couchbase-powerbi-connector/1.0/couchbase-powerbi-connector-1.0.mez[Power BI Connector Download]
+
+The supported and tested dependencies of this release are:
+
+* https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-1.1-win64.msi[couchbase-odbc-1.1-win64.msi]
+
+** See the https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-driver-1.1-notices.txt[Notices File] for dependency licenses for the ODBC Driver.
 
 [#v-1-0]
 == Version 1.0 (2024-01-25)
```
Comment (janhavitripurwar) [inline modules/ROOT/pages/release-notes.adoc:27]: If we keep version 1.0, should we specify that it only works with OpenSSL 1.1.1?

**Review by janhavitripurwar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 4 above)
Comment (RayOffiah) [reply modules/ROOT/pages/release-notes.adoc:27]: Yes, I should probably note OpenSSL dependencies here.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (RayOffiah) [reply modules/ROOT/pages/index.adoc]: Hold on

**Review by janhavitripurwar — APPROVED** [review]

**Pull request merged**